### DISCUSSION TITLE
Is it possible to change the .output chunk name strategy?

### DISCUSSION THREAD

**Discussion by Aareksio** [nuxt/nuxt#17905] [Questions]
Hello! Thank you for amazing work!

### TL;DR: **Can we easily change chunk naming strategy to webpack-like `[index]-[hash].mjs`?**

---

By default, nuxt 3 production build (`nuxt build`) seemingly produces a _chunk_ for each component. Those files are located and served from `.output/public/_nuxt` directory. For example:

```
entry-XXXXXXXX.mjs
footer-XXXXXXXX.mjs
header-XXXXXXXX.mjs
index-XXXXXXXX.mjs
```

We ran into an issue with **ad blocking** extensions blocking our chunk names due to regex patterns, this chunk was blocked:

```
placeholder-ad-XXXXXXXX.mjs
```

Failure to load single chunk out of the bundle causes the whole app to bail.

At first we thought avoiding ad-related names is enough, but then a chunk named `footer-social-XXXXXXXX.mjs` caused our app to fail for someone who claimed they "do not have any non-standard list enabled". While not an issue in nuxt, this is a major inconvenience. In even the moderate app we should expect hundreds of components, each with unique name. We can never be sure a seemingly safe word is not related to advertisement (or otherwise invasive practice) in some language.

---

_With the backstory behind,_
**can we easily change chunk naming strategy to webpack-like `[index]-[hash].mjs`?**

---

I am aware it is possible supply custom `config.vite.build.rollupOptions.output.chunkFileNames` function, but this seems overly complicated for such critical feature. What are your thoughts? 

Personally I'd go as far as suggesting index-based strategy should be default - a lot of users may be unaware of this issue, having their website blocked for huge segments of audience (eg. country / language specific list). It would also be beneficial to have default error handled for chunk-related issues.
